OrdersApplication/OrdersSectionCell/OrdersSectionCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  OrdersSectionCell.swift
+//  OrdersApplication
+//
+//  Created by Esraa Khaled   on 23/09/2022.
+//
+
+import UIKit
+
+class OrdersSectionCell: UITableViewCell {
+
+    @IBOutlet weak var orderImage: UIImageView!
+    @IBOutlet weak var arrowImage: UIImageView!
+    @IBOutlet weak var orderDate: UILabel!
+    @IBOutlet weak var orderTime: UILabel!
+    @IBOutlet weak var totalPrice: UILabel!
+    @IBOutlet weak var orderNum: UILabel!
+    @IBOutlet weak var view: UIView!
+    @IBOutlet weak var parentView: UIView!
+    
+    var pressed: (()->())?
```

Comment:
```suggestion
    private var pressed: (()->())?
```

```swift
func onPressed(_ pressed: () -> Void) { 
    self.pressed = pressed
}
```

To avoid direct updates to pressed outside the class.